tests/markdown/MarkdownBearTest.py

```diff
@@ -29,3 +45,22 @@
     valid_files=(test_file2,),
     invalid_files=(test_file3,),
     settings={'max_line_length': 10})
+
+MarkdownBearValidateLinksTest = verify_local_bear(
+    MarkdownBear,
+    valid_files=(test_file5,),
+    invalid_files=(test_file4,))
+
+
+@generate_skip_decorator(MarkdownBear)
+class MarkdownBearValidateLinksMessageTest(unittest.TestCase):
+
+    def setUp(self):
+        self.uut = MarkdownBear(Section('name'), Queue())
+
+    def test_validate_links_message(self):
+        content = test_file4.splitlines()
```

Comment:
I am already checking it in the MarkdownValidateLinksTest, this is to check if the message of the result is correct.